index.html

```diff
@@ -6,6 +6,7 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
+<h1>Dit is een titel</h1>
 <p>Dit is wijziging 1</p>
 <p><a>Dit is wijziging 2</a><p>
```

Comment:
Misschien is hier een <section>Waroom dit zo een mooie opdracht is?</section> nog handig ?

Reply:
Ik bedoelde een section ;)